### PR TITLE
Update unifi.markdown

### PR DESCRIPTION
**Pull request by jordaosoft**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.
- [x] Add missing documentation. 
## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifi.markdown [hunk 1]
```diff
@@ -39,7 +39,7 @@ The UniFi controller allows you to create multiple users on it besides the main
 
 ### UniFi OS
 
-For UniFi OS a local-only user needs to be created. A user who uses the Ubiquiti cloud will not work. You can do this in the manage users section on the UniFi OS dashboard. Make sure to give it the right permissions for the functions you want to use. 
+For UniFi OS a local-only user needs to be created. A user who uses the Ubiquiti cloud will not work. You can do this in the manage users section on the UniFi OS dashboard. Make sure to give it the right permissions for the functions you want to use. Note the Dream Machine Pro needs the port to be 443.
```
Comment (frenck) [inline source/_integrations/unifi.markdown:42]: This is a bit of a weird place to make a note about the dream machine?

Isn't the part of the configuration instructions?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/unifi.markdown (hunk 1 above)
Comment (jordaosoft) [reply source/_integrations/unifi.markdown:42]: Sorry if this is the wrong place but the  Dream Machine pro (that runs UniFi OS) needs the port to be 443 as the default does not work. I did not see this anywhere in the configuration instructions.  I spent over an hour thinking I had the wrong user/password until I found a mention in the forum about the port.

Comment (frenck) [reply source/_integrations/unifi.markdown:42]: ✅ Thanks 👍

**Review by frenck — APPROVED** [review]
Thanks, @jordaosoft 👍

**Pull request merged**